### PR TITLE
fixed default and override member sig creation

### PR DESCRIPTION
**Pull request by Martin521**

### CODE REVIEW

File: tests/Telplin.Core.Tests/TypeTests.fs [hunk 1]
```diff
@@ -98,6 +98,53 @@ type Foo =
     abstract member Bar: x: int -> y: int -> int
 """
 
+[<Test>]
+let ``Abstract member with default implementation`` () =
```
Comment (nojaf) [inline tests/Telplin.Core.Tests/TypeTests.fs:102]: Could you please lowercase the test names?
See [docs](https://nojaf.com/telplin/docs/contributing.html#Pull-request-ground-rules).

I tried to do it myself but I can't seem to push it to your branch.

**Review by nojaf — CHANGES_REQUESTED** [review]
Thank you! Good catch!

File: tests/Telplin.Core.Tests/TypeTests.fs (hunk 1 above)
Comment (Martin521) [reply tests/Telplin.Core.Tests/TypeTests.fs:102]: Sure, I will.

**Review by nojaf — APPROVED** [review]
Many thanks!

**Pull request merged**